### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by milen-ale**
- In this pull request I managed To do the following

- Create a table named `owners` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `full_name`: string
  - `age`: integer
  
trans1

- Create a table named `species` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `name`: string
  
trans2

- Modify `animals` table:
  - Make sure that id is set as autoincremented PRIMARY KEY
  trans3
trans4

  - Remove column `species`
  
transe5

  - Add column `species_id` which is a foreign key referencing `species` table
  
trans6

  - Add column `owner_id` which is a foreign key referencing the `owners` table
  
trans7

- Insert the  data into the `owners` table:
trans8

trans8
pok
transe10

-Write queries (using `JOIN`) to answer the following questions

transq 11

transe q13

transe q14

-How many animals are there per species?

-List all Digimon owned by Jennifer Orwell

-List all animals owned by Dean Winchester that haven't tried to escape.

-Who owns the most animals?

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,103 @@
...
+
+SELECT animals.name, owners.full_name FROM owners INNER JOIN animals ON animals.owner_id = owners.id WHERE owners.full_name = 'Jennifer Orwell';
+
+SELECT animals.name, owners.full_name FROM owners INNER JOIN animals ON animals.owner_id = owners.id WHERE owners.full_name = 'Bob';
+
+SELECT animals.name, owners.full_name FROM owners INNER JOIN animals ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name, owners.full_name FROM owners INNER JOIN animals ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Winchester';
+
+SELECT animals.name, owners.full_name FROM owners INNER JOIN animals ON animals.owner_id = owners.id WHERE owners.full_name = 'Jodie Whittaker';
```
Comment (DeliceLydia) [inline queries.sql:103]: - [ ] I can't see these queries
-  List all owners and their animals, remember to include those that don't own any animal.
-  How many animals are there per species?

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @milen-ale, 

Good job so far! there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: queries.sql (hunk 1 above)
Comment (rloterh) [reply queries.sql:103]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Milen,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾  Congratulations!  🎉💯🌟 

## Highlights 👍
- Queries properly implemented

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- You used incorrect gitflow/ GitHub flow for this project. Kindly note that the merge request for this project should flow from **from feature into development**. Kindly note this important step for subsequent projects.
- This project appears to be worked on by a sole contributor, therefore kindly update the Author section of the ReadMe accordingly and change the header title _Authors_ to _Author_. 
- Generally, the subheading text Author1 also should be replaced with your full name in your ReadMe.
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**